### PR TITLE
Patch release

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/crisprseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/crisprseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.2.1dev]
```
Comment (LaurenceKuhl) [inline CHANGELOG.md:6]:
```suggestion
## [v2.2.1 - Romarin Curie - patch]
```

File: nextflow.config [hunk 2]
```diff
@@ -285,7 +285,7 @@ manifest {
     description     = """Pipeline for the analysis of CRISPR data"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '2.2.0'
+    version         = '2.3.0dev'
```
Comment (mirpedrol) [inline nextflow.config:288]: this version should be `2.2.1`

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 report_comment: >
 
-  This report has been generated by the <a href="https://github.com/nf-core/crisprseq/releases/tag/2.2.0" target="_blank">nf-core/crisprseq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/crisprseq/tree/dev" target="_blank">nf-core/crisprseq</a>
```
Comment (mirpedrol) [inline assets/multiqc_config.yml:3]: Should be `releases/tag/2.2.1`

File: assets/multiqc_config.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/crisprseq/2.2.0/docs/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/crisprseq/dev/docs/output" target="_blank">documentation</a>.
```
Comment (mirpedrol) [inline assets/multiqc_config.yml:5]: Should be `releases/tag/2.2.1`

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.2.1 Romarin Curie - patch](https://github.com/nf-core/crisprseq/pull/174)
```
Comment (mirpedrol) [inline CHANGELOG.md:6]: Sorry I missed this link in the previous PR :) 
```suggestion
## [v2.2.1 Romarin Curie - patch](https://github.com/nf-core/crisprseq/releases/tag/2.2.1)
```

Also a reminder that we should add the date before merging this PR

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.2.1 Romarin Curie - patch](https://github.com/nf-core/crisprseq/releases/tag/2.2.1) - [23.07.2024]
+
+### Fixed
+
+- Fix singularity image pull tag for MAGeCKFlute ([#160](https://github.com/nf-core/crisprseq/pull/160))
+- Skip dolar signs on containerOptions ([#163](https://github.com/nf-core/crisprseq/pull/163))
```
Comment (mashehu) [inline CHANGELOG.md:11]:
```suggestion
- Escape dollar signs in `containerOptions` ([#163](https://github.com/nf-core/crisprseq/pull/163))
```

**Review by mashehu — APPROVED** [review]

**Review by JoseEspinosa — APPROVED** [review]
LGTM 🚀

**Pull request merged**